cEP-0009.md

```diff
@@ -0,0 +1,181 @@
+| Metadata     |                                                          |
+| ------------ |----------------------------------------------------------|
+| cEP          | 9                                                        |
+| Version      | 0.1                                                      |
+| Title        | Extract and utilize information from project files       |
```

Comment:
It's not for me. I don't know if it's my font messing up.

![img](https://cloud.githubusercontent.com/assets/7521600/26461355/8e2ca4cc-419a-11e7-9d4f-9b792cd09f60.png)